src/IssueViz/Editor/TaggableBufferIndicator.cs

```diff
@@ -40,14 +54,7 @@ public bool IsTaggable(ITextBuffer buffer)
                 return false;
             }
 
-            var isOutputBuffer = buffer.ContentType.IsOfType("Output");
```

Comment:
Unfortunately this check was not enough.

When you first open a tool window in VS, VS creates behind the scenes a temp file called "Temp.txt". The content type of this file is whatever window you opened. I previously thought it happens just with output windows, hence the "IsOfType(Output)", but apparently it can happen for any tool window. So for example, if you open the command window, it will create a temp.txt with content type "Command". 

I tried to see if there is anything to identify these windows but there is nothing, and their content type inherits from "text", which we should analyze because we support secrets.

So in the end I opted for checking if it really exists or not, and that seems to work.